test/spec/modules/vidazooBidAdapter_spec.js

```diff
@@ -129,7 +132,7 @@ describe('VidazooBidAdapter', function () {
       expect(requests).to.have.length(1);
       expect(requests[0]).to.deep.equal({
         method: 'POST',
-        url: `${URL}/prebid/multi/59db6b3b4ffaa70004f45cdc`,
+        url: `${createDomain(SUB_DOMAIN)}/prebid/multi/59db6b3b4ffaa70004f45cdc`,
```

Comment:
Should this be `SUB_DOMAIN`?  This test should be failing for you.  Otherwise, LGTM